libs/SalesforceSDK/res/values/sf__styles.xml

```diff
@@ -2,7 +2,9 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <style name="SalesforceSDK" parent="Theme.AppCompat.DayNight">
+    <style name="SalesforceSDK" parent="Theme.AppCompat.Light.DarkActionBar">
+        <item name="colorPrimary">@color/sf__primary_color</item>
```

Comment:
AppCompat picks up these two colors from a different attribute set, so I is necessary to have both `colorPrimary` and `android:colorPrimary`.